demo/src/screens/componentScreens/FeatureHighlightScreen.js

```diff
@@ -7,47 +8,84 @@ class FeatureHighlightScreen extends Component {
     super(props);
     this.closeHighlight = this.closeHighlight.bind(this);
     this.showHighlight = this.showHighlight.bind(this);
+    this.onRefUpdated = this.onRefUpdated.bind(this);
+
+    this.state = {
+      showFTE: false,
+      fteHighlightViewTag: undefined,
```

Comment:
what is fteHighlightViewTag for, are you using it anywhere?

Reply:
No, you're right. This is leftover from Artal's fix suggestion. I will remove it.